documentation/sphinx/source/special-keys.rst

```diff
@@ -275,7 +275,7 @@ Deprecated Keys
 
 Listed below are the special keys that have been deprecated. Special key(s) will no longer be accessible when the client specifies an API version equal to or larger than the version where they were deprecated. Clients specifying older API versions will be able to continue using the deprecated key(s).
 
-#. ``\xff\xff/management/profiling/<client_txn_sample_rate|client_txn_size_limit>`` Deprecated as of API version 710300. The corresponding functionalities are now covered by the global configuration module. For details, see :doc:`global-configuration`. Read/write. Changing these two keys will change the corresponding system keys ``\xff\x02/fdbClientInfo/<client_txn_sample_rate|client_txn_size_limit>``, respectively. The value of ``\xff\xff/management/client_txn_sample_rate`` is a literal text of ``double``, and the value of ``\xff\xff/management/client_txn_size_limit`` is a literal text of ``int64_t``. A special value ``default`` can be set to or read from these two keys, representing the client profiling is disabled. In addition, ``clear`` in this range is not allowed. For more details, see help text of ``fdbcli`` command ``profile client``.
+#. ``\xff\xff/management/profiling/<client_txn_sample_rate|client_txn_size_limit>`` Deprecated as of API version 720. The corresponding functionalities are now covered by the global configuration module. For details, see :doc:`global-configuration`. Read/write. Changing these two keys will change the corresponding system keys ``\xff\x02/fdbClientInfo/<client_txn_sample_rate|client_txn_size_limit>``, respectively. The value of ``\xff\xff/management/client_txn_sample_rate`` is a literal text of ``double``, and the value of ``\xff\xff/management/client_txn_size_limit`` is a literal text of ``int64_t``. A special value ``default`` can be set to or read from these two keys, representing the client profiling is disabled. In addition, ``clear`` in this range is not allowed. For more details, see help text of ``fdbcli`` command ``profile client``.
```

Comment:
This was deprecated in Apple version 720, so we should certainly not say deprecated as of 710300. You're right that we should ideally refer to the equivalent Snowflake API version where it was deprecated. However, since we don't have a one-to-one mapping, I checked [this file in the snowflake/release-71.2 branch](https://github.com/apple/foundationdb/blob/snowflake/release-71.2/documentation/sphinx/source/special-keys.rst), and it refers to API version 720 there, so I stuck with the same to avoid any confusion. PLMK what version you think would be best to have here instead.

cc @sfc-gh-jslocum

Reply:
I think we should refer to Snowflake API versions. It means it should be either 710300 or 710200 here. 

I see the these keys were removed by the commit c0264a8522023fda3f814c503a5606e05d5364d4, and this commit is included into the 71.2 release, so 710200 would be correct here.

Reply:
Done